### ISSUE TITLE
Cover: Focal point picker label is not announced by screen readers

### ISSUE THREAD

**Issue by carolinan** [WordPress/gutenberg#50991]
### Description

The cover block has a setting in the sidebar called focal point picker, where you can select a left and top numeric value.

When navigating the settings sidebar with a screen reader, the label "Focal point picker" is not announced, so the settings are read out like:
Media settings, expanded, button
Fixed background, unchecked checkbox
Repeated background, unchecked checkbox
50, left, stepper
50, top, stepper
Alternative text

It is unclear what the stepper is, there is no context.

### Step-by-step reproduction instructions

Activate your screen reader. I tested with VoiceOver on mac.

In the block editor, add a cover block.
Navigate to the block settings sidebar, and tab through the settings tab.
Try to understand what the steppers are used for.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by annezazu** [comment]
Thanks for opening this! Was about to track down help to fix but I see you've already handled that here https://github.com/WordPress/gutenberg/pull/50993 Closing out as a result. Nice work.
